Source/WebCore/platform/graphics/ColorMatrix.h

```diff
@@ -50,12 +55,24 @@ class ColorMatrix {
         return m_matrix[(row * ColumnCount) + column];
     }
 
+    const std::array<float, 20>& data() const { return m_matrix; }
```

Comment:
```suggestion
    std::span<const float, RowCount * ColumnCount> data() const { return m_matrix; }
```

I am not entirely sure if the conversion to span here works without and fiddling. But it would be good to avoid leaking the std::array detail and using `RowCount * ColumnCount` instead of 20.